### PR TITLE
ShEx component

### PR DESCRIPTION
**Pull request by glenna-m**

### CODE REVIEW

**Comment by catch-point** [comment]
LGTM

File: components/shex-cmumps-to-rdf.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+/**
+ * File:  shex-cmumps-to-rdf.js
```
Comment (dbooth-boston) [inline components/shex-cmumps-to-rdf.js]: Please add an opening comment saying overall what this does.

File: components/shex-cmumps-to-rdf.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+var graphContext = {
+    "@context": {
+        "loinc": "http://hokukahu.com/schema/loinc#",
+        "hptc": "http://hokukahu.com/schema/hptc#",
+        "cpt": "http://hokukahu.com/schema/cpt#",
+        "ndc": "http://hokukahu.com/schema/ndc#",
+        "icd9cm": "http://hokukahu.com/schema/icd9cm#",
+        "npi": "http://hokukahu.com/schema/npi#",
+        "nddf": "http://hokukahu.com/schema/nddf#",
+        "@vocab": "http://hokukahu.com/schema/cmumpss#",
```
Comment (dbooth-boston) [inline components/shex-cmumps-to-rdf.js:130]: Will this work with both cmumps and chcs input?  If so, how?

**Comment by dbooth-boston** [comment]
LGTM subject to one minor request.  No need for re-review.

**Pull request closed without merging**